template/Cake.Frosting.Template.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageId>Cake.Frosting.Template</PackageId>
+    <Title>Cake.Frosting templates for the .NET SDK.</Title>
+    <Description>Cake.Frosting templates for the .NET SDK.</Description>
+    <Authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström and contributors</Authors>
+    <Copyright>Copyright (c) .NET Foundation and contributors</Copyright>
+    <PackageTags>Cake;Script;Build</PackageTags>
+    <PackageIcon>cake-medium.png</PackageIcon>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/cake-build/frosting</RepositoryUrl>
```

Comment:
This should have .git at the end of it.

Reply:
Should this be corrected too then?
https://github.com/cake-build/cake/blob/daa3c4f26a5d54cbb8c0b65301eca098d2703f01/src/Shared.msbuild#L10

Reply:
Yes, I would say so.  Having .git at the end of the URL is the recommendation that came from NuGet Team: https://twitter.com/adgrv/status/1127591787697061888

Reply:
Corrected and moved to Shared.msbuild 
![image](https://user-images.githubusercontent.com/1647294/86858739-89b84f00-c0c1-11ea-9079-fda5a0e0cf70.png)